io/genbank/genbank.go

```diff
@@ -66,14 +68,24 @@ type Meta struct {
 
 // Feature holds the information for a feature in a Genbank file and other annotated sequence files.
 type Feature struct {
-	Type                 string            `json:"type"`
-	Description          string            `json:"description"`
-	Attributes           map[string]string `json:"attributes"`
-	SequenceHash         string            `json:"sequence_hash"`
-	SequenceHashFunction string            `json:"hash_function"`
-	Sequence             string            `json:"sequence"`
-	Location             Location          `json:"location"`
-	ParentSequence       *Genbank          `json:"-"`
+	Type                 string
+	Description          string
+	Attributes           multimap.MultiMap[string, string]
```

Comment:
Why use multimap vs map[string][]string?

Reply:
switched to use map[string][]string